traits/src/nft.rs

```diff
@@ -20,10 +17,6 @@ pub trait NFT<AccountId> {
 	/// The number of NFTs assigned to `who`.
 	fn balance(who: &AccountId) -> Self::Balance;
 
-	/// The owner of the given token ID. Returns `None` if the token does not
-	/// exist.
-	fn owner(token: (Self::ClassId, Self::TokenId)) -> Option<AccountId>;
```

Comment:
why remove those?

Reply:
The Inspect trait contains a "owner" function and the "transfer" trait contains "transfer". If we keep both, it introduces extra confusion. So I thought it would be more "clean" to remove them so there's only one transfer() and one owner()

Reply:
In that case do we need `balance`? And should just rename this to something like `InspectExtended`

Reply:
yeah good idea. I'll look into this

Reply:
There's no use case for "balance" right now, but I kept it in case if it is needed in the future.